scripts/data/git/pullrequests.ts

```diff
@@ -50,6 +57,31 @@ function getPullRequestsForRepository(username: string, repoId: string, skip = 0
     });
 }
 
+function getReviewedPullRequestsForRepository(username: string, repoId: string, skip = 0): Promise<GitPullRequest[]> {
+    const criteria = {
+        reviewerId: username,
+        status: PullRequestStatus.All,
+    } as GitPullRequestSearchCriteria;
```

Comment:
Pass in the criteria so that ```getPullRequestsForRepository``` does not have to be duplicated.

Reply:
Done. I also renamed the other method getPullRequests to => getCreatedPullRequests to be consistent with getReviewedPullRequests.